src/components/browse/components/above-table-controls/RightButtonsSection.js

```diff
@@ -29,7 +29,7 @@ export const ConfigureVisibleColumnsButton = React.memo(function ConfigureVisibl
     );
 });
 ConfigureVisibleColumnsButton.defaultProps = {
-    "className" : "btn btn-outline-primary"
+    "className" : "control visiblity btn"
```

Comment:
possible typo for `visiblity?

Reply:
Thank you for catching this Utku! There is no use for this class at the moment actually, so I will remove.